rust/Anchor.toml

```diff
@@ -27,8 +30,7 @@ staking = "stk9HFnKhZN2PZjnn5C4wTzmeiAEgsDkbqnHkNjX1Z4"
 test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
 
 [workspace]
-members = ["item", "namespace", "matches", "staking"]
-types = "../js/lib/src/idls/"
```

Comment:
Removed this line because it was creating idl files automatically to this path.
But since anchor does not support imported struts in its idl, I was managing staking idl manually.
So it messes up some program idls.

Reply:
I believe this will cause trouble for the other packages because now their idls will grow stale.

Reply:
Unless you have another idea for how we should manage idls, I'm all ears

Reply:
The idls are exported to `target/idl` directory by default. Then we can manually update idls in js lib if there are changes.

Reply:
The idls of item, player, staking should not be overwritten by default because we're managing theses idls manually.

Reply:
I kinda liked them being exported to the js lib which is checked into git so it's easier to see what changed, within the target directory that is not in git I think

Reply:
So we can package the IDL with the code, also if you do want to make this change you probably should take out that directory and not leave it right?

Reply:
I wasn't aware especially that we were managing the item IDL manually lol, can you point me to those manual changes? I think it's just been the anchor generation

Reply:
Oh, man, after your change, I should rollback the staking idl in `js/lib` to the original one at every `anchor build`. I believe `item` and `player` are in the same situation. This is very troublesome.

Reply:
We talked about this on tg and k suggested we don't rely on manually editing the idls

Reply:
Good idea.
I will add back types config first, and then we can do it as a separate PR I think.